examples/accounts/accounts.repl

```diff
@@ -46,18 +46,19 @@
 
 ; initiate pact with normal execution
 (payment "123" "us" "456" "them" 12.0 (time "2016-07-22T11:26:35Z"))
+(pact-state)
 (expect "balance of 123 after debit step" 217.0 (with-read accounts "123" { "balance" := b } b))
 
 ; test rollback
-(continue-pact 6 0 true)
+(continue-pact 0 true "Tug6q-wmaru1QqUMjm7RiwhnsQMTRcMHus65HwF7laE")
```

Comment:
Getting rid of pact-id for private pact example, but need to keep for the public pact ones. This is because omitting the pact-id assumes that there's a PactExec in the environment, which is only the case if the pact is initialized in the same transaction. Therefore, the pact-id needs to be kept for examples where the pact is initialized in a different transaction (i.e. the public pact example). Resolving.